src/app/components/MediaLoader/utils/buildSettings.ts

```diff
@@ -36,7 +37,6 @@ const buildSettings = ({
     enableToucan: true,
     appType: isAmp ? 'amp' : 'responsive',
     appName: service !== 'news' ? `news-${service}` : 'news',
-    externalEmbedUrl: '', // TODO: Check requirements on this, will need added in future when media player has dedicated page for AMP support
```

Comment:
Why don't we keep this and set it to `embedUrl`? I know that this will only be used/displayed in the SMP UI if embedding is permitted, but seems like there would be no harm in setting it here? 

I'm thinking about our future cases, when we migrate the other media pages to use the Media Loader - it would be good if this could be defaulted to a value. Although @amoore108 mentioned in https://github.com/bbc/simorgh/pull/11917/files#r1734966810 about there being more duplication... not sure - we're happy to follow the same patterns.

Reply:
If it can be set to a value all the time then I see no harm in that. I'd be inclined to use https://github.com/bbc/simorgh/blob/a86bab4de17c0a5ecf9ae65743fba862b79dedda/ws-nextjs-app/utilities/buildAvEmbedUrl/index.ts#L11 directly in here rather than as a prop. May need some tweaking to pass in the right arguments, but should be doable.

Reply:
We also have https://github.com/bbc/simorgh/blob/f7e2b2e6c89e0fce7f8ba2d681fa45f61df41f20/src/app/lib/utilities/getUrlHelpers/getEmbedUrl/index.js - I am wondering if they could be consolidated? (This function is still used by the legacy media player to construct the URL for AMP). 

Whichever function we use, could we move it to a shared location in Simorgh (rather than within the NextJS app?)

Reply:
Yea should ideally standardise on one. Slightly hesitant to touch the AMP related work until we start working on it, which is soon anyway, but that function may be the one we go with and refactor for both syndication and AMP url generation.

Reply:
Makes sense thanks! I will refactor those functions.

Reply:
I've had a go refactoring. I've added some comments.